tests/integration/standard/test_tls_resumption.py

```
@@ -0,0 +1,366 @@
+        ssl1.close()
+        if session is None:
+            return False
+        time.sleep(0.1)
+        s2 = socket.create_connection((host, port), timeout=5)
+        ssl2 = ctx.wrap_socket(s2, do_handshake_on_connect=False)
+        ssl2.session = session
+        ssl2.do_handshake()
+        reused = ssl2.session_reused
+        ssl2.close()
+        return bool(reused)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)